src/components/text-input/text-input.tsx

```diff
@@ -136,8 +137,10 @@ export const TextInput = ({
           {error}
         </Paragraph>
       )}
-      {help && (
-        <Paragraph paragraphSize="small" marginTop="8px">
+      {React.isValidElement(help) ? (
```

Comment:
importaljuk ugy be, hogy ne kelljen a `React` namespace ele, mint mondjuk a `FunctionComponent`